python/pylibcudf/pylibcudf/tests/test_column_from_array.py

```diff
@@ -40,66 +40,83 @@
 ]
 
 
-@pytest.fixture(params=NUMPY_DTYPES, ids=repr)
-def np_1darray(request):
-    dtype = request.param
-    return np.array([0, 1, 2, 3], dtype=dtype)
-
-
-@pytest.fixture(params=NUMPY_DTYPES, ids=repr)
-def np_2darray(request):
-    dtype = request.param
-    return np.array([[0, 1, 2], [3, 4, 5]], dtype=dtype)
+@pytest.fixture(params=[1, 2, 3, 4], ids=lambda x: f"ndim={x}")
+def shape(request):
```

Comment:
Do we have a test with a 0-dim array (e.g. `np.array(0)`)? I think we would/should raise in this case?

Reply:
Added a test